src/connection/buffer.hpp

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright 2008 Search Solution Corporation
```

Comment:
nit: 새로 생성한 파일들은 이 문구를 포함하지 않고 제거해주시면 됩니다.
```suggestion
 *
```